time stamp
uint16 id
string name
int16[] data
